tests/test_resolver.py

```diff
@@ -26,3 +50,99 @@ def test_open_stage_and_logging(capfd):
     assert "_OpenAsset" in outputs[6]
     assert "_GetModificationTimestamp" in outputs[7]
     assert "_GetExtension" in outputs[8]
+
+
+# As all the data tends to follow the same form, convenience method
+# to avoid repeating myself
+def assert_parking_lot_structure(usd_stage):
+    floor = usd_stage.GetPrimAtPath("/ParkingLot/ParkingLot_Floor_1")
+    car1 = usd_stage.GetPrimAtPath("/ParkingLot/ParkingLot_Floor_1/Car1")
+    car2 = floor.GetChild("Car2")
```

Comment:
[question] Any particular reason to split the lookup approach? The second form seems easier to maintain.

Reply:
It's a vague notion, but I thought to do this for the parent assetized/child non assetized test, just in case USD does some under-the-hood things where it can bypass resolving the intermediate file. It's just a small flourish really, will remove if you think it's any maintenance burden.

Reply:
Id lean towards the simpler/consistent version

Reply:
https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/2aad5788a6802d7074bc48fc8698d104ce777038

Reply:
Possibly based on Davids comments below we'll remove the duplicate car entries now, since they're accessed as exact duplicates.

Reply:
I kind of like there being two as it means were exercising multiple resolves of the same path